### PR TITLE
added pristine zen garden html

### PR DESCRIPTION
**Pull request by annika-philipp**
Tidied html file

### CODE REVIEW

File: pristine-zen-garden.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (jessicaantiope) [inline pristine-zen-garden.html]: The head tag and its contents are not indented properly.  If you indented the head tag first and then the rest by one that would be cool

File: pristine-zen-garden.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            <p>This is a learning exercise as well as a demonstration. You retain full copyright on your graphics (with limited exceptions, see <a href="http://www.mezzoblue.com/zengarden/submit/guidelines/">submission guidelines</a>), but we ask you release your <abbr title="Cascading Style Sheets">CSS</abbr> under a Creative Commons license identical to the <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/" title="View the Zen Garden's license information.">one on this site</a> so that others may learn from your work.</p>
+            <p role="contentinfo">By <a href="http://www.mezzoblue.com/">Dave Shea</a>. Bandwidth graciously donated by <a href="http://www.mediatemple.net/">mediatemple</a>. Now available: <a href="http://www.amazon.com/exec/obidos/ASIN/0321303474/mezzoblue-20/">Zen Garden, the book</a>.</p>
+        </div>
+
+        <footer>
+            <a href="http://validator.w3.org/check/referer" title="Check the validity of this site&#8217;s HTML" class="zen-validate-html">HTML</a>
+                <a href="http://jigsaw.w3.org/css-validator/check/referer" title="Check the validity of this site&#8217;s CSS" class="zen-validate-css">CSS</a>
```
Comment (jessicaantiope) [inline pristine-zen-garden.html]: I think this <a> need to be one tab back

File: pristine-zen-garden.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+<meta name="robots" content="all">
+
+</head>
+
+<body id="css-zen-garden">
+  <div class="page-wrapper">
+    <section class="intro" id="zen-intro">
+
+        <header role="banner">
+            <h1>CSS Zen Garden</h1>
```
Comment (Steve3029) [inline pristine-zen-garden.html]: I think the indentation in a page should be uniform. The indentations of previous level were two spaces. But begin from header tag, the indentations were 4 spaces.

Comment (annika-philipp) [reply pristine-zen-garden.html]: Hi Steve, thanks for checking! But the header, h1 etc. all sit within the section tab which closes on line 37 :)

**Pull request merged**